arelle/plugin/validate/HMRC/__init__.py

```diff
@@ -165,10 +165,7 @@ def validateXbrlStart(val, parameters=None, *args, **kwargs):
         val.isAccounts = not val.isComputation
 
     val.txmyType = None
-    for doc in val.modelXbrl.modelDocument.referencesDocument:
-        ns = doc.targetNamespace
-        if not ns:
-            continue
+    for ns in val.modelXbrl.prefixedNamespaces.values():
```

Comment:
Out of curiosity, why does this fix the issue?

Reply:
targetNamespaces are not set when loading from IXDS because IXDS handles them. So we need a more direct way of getting the namespaces rather than targetNamepsace.

Reply:
Discussed offline